plugins/woocommerce/src/Internal/Admin/Orders/PageController.php

```diff
@@ -136,7 +136,7 @@ public function setup(): void {
 		if ( 'admin_menu' === current_action() ) {
 			$this->register_menu();
 		} else {
-			add_action( 'admin_menu', 'register_menu', 9 );
+			add_action( 'admin_menu', array( $this, 'register_menu' ), 9 );
```

Comment:
Huh, and this never threw an error before? Which makes me wonder if this ever gets invoked?